charms/kserve-controller/lib/charms/resource_dispatcher/v0/resource_dispatcher.py

```diff
@@ -134,7 +134,7 @@ class KubernetesManifest:
 
     def __post_init__(self):
         """Validate that the manifest content is a valid YAML."""
-        self.manifest = yaml.safe_load(self.manifest_content)
+        self. manifest = yaml.safe_load(self.manifest_content)
```

Comment:
looks like this was changed in https://github.com/canonical/resource-dispatcher/pull/45, I think we need to re-publish the library and fetch it once again